packages/harpguru-core/src/components/harp-face/hooks/use-octave-column-groups/use-octave-column-groups.ts

```diff
@@ -1,37 +1,24 @@
-import type { HarpStrata } from 'harpstrata'
+import { useGlobal } from 'reactn'
 import type { Degree, HarpFaceMatrix } from 'harpparts'
 
-import { transposeMatrix } from '../../packages/transpose-matrix'
+import { transposeMatrix } from '../../../../packages/transpose-matrix'
 
 import { getOctaveColumnGroups } from './get-octave-column-groups'
 import type { ColumnRanges } from './get-octave-column-groups'
 import { arrayHasRoot } from './array-has-root'
 
-type HarpFaceFacts = {
-  readonly rowCount: number
-  readonly columnCount: number
-  readonly octaveColumnGroups: ColumnRanges
-}
-
-export const getHarpFaceFacts = (
-  activeHarpStrata: HarpStrata
-): HarpFaceFacts => {
+export const useOctaveColumnGroups = (): ColumnRanges => {
```

Comment:
This and the `getOctaveColumnGroups` function can probably be improved in naming and test orientation (ie test the iterator callback, not the outer function)

Reply:
Raised as techdebt in #113 

The work done to make the app sizing dynamic (at #107) was used as a platform for additional speculative pull requests to ensure that this was the correct direction to start down.

Rather than addressing the finer details in this or subsequent pull request and potentially struggling to merge up to those subsequent branches I will prefer to record the issues in each merge request and resolve them afterwards.

This has a limit of course. Major issues will not be merged to master. I will need to find a way around that if it happens.